src/main/java/com/tradeshift/HelloWorld.java

```diff
@@ -0,0 +1,47 @@
+package com.tradeshift;
+
+import java.net.HttpURLConnection;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.*;
+
+import com.tradeshift.service.HelloWorldInterface;
+
+
+@Path("/")
+@Controller
+public class  HelloWorld {
```

Comment:
We generally go by (and the rules also state) that this should be called `HelloWorldResource`